### ISSUE TITLE
Map size as a function of phase

### ISSUE THREAD

**Issue by maclariz** [pyxem/orix#426]
Just found that mapping a two phase material, the map extent is different if I map the majority phase - it covers the full scan area - to the minority - which only covers the area containing this phase.  Thus the IPF maps have different sizes and this makes comparison hard.  Any way round this?

**Comment by hakonanes** [comment]
Do you want to combine the maps into one array? Or save images of different IPF maps but with the same shape?

One way to do the former is shown in the [crystal map tutorial](https://orix.readthedocs.io/en/stable/tutorials/crystal_map.html#Plotting) in cell number 61 (the color key is not important here).

To do the latter, I suggest to do a variant of the example in the tutorial, where you only fill the map points of one phase and pass the whole array to the whole crystal map plot method, i.e. not to the crystal map of one phase. Does that work for you?

**Comment by maclariz** [comment]
Thanks @hakonanes .  After various coding stupidities, I got this to work...

**Comment by hakonanes** [comment]
Glad to hear it! Were any of my suggestions useful, or did you do something else? It might be an idea to turn your case into an example in the examples gallery.

Also, I wonder if it is possible to allow a `full_shape` parameter to `CrystalMap.plot()`, because the original shape after slicing (e.g. with `CrystalMap["phase1"]`) is still kept for internal use.

**Comment by maclariz** [comment]
I basically did as per your suggestions, except:

- I am mapping IPF x, y and z
- Thus generated a list of 3 correct shape arrays of zeros
- Then filled the appropriate spots for the second phase with the colours already defined.  
- Then plotted all three IPF maps using the whole map to define the area and these colour arrays to define the colours

As for making an example, my thoughts are "not yet" until the work is published.  Anything that gets published using orix could also be made into an example.  The first will be more of a methods paper about mapping of 2-phase titanium, doing crystallography with the results, and getting quantitatively interpretable results, which fit crystallographic theory, all with few nm step size, which is unprecedented for such 2-phase Ti alloys (which are typically studied with EPSD at much coarses step size, which is therefore insensitive to the very fine scale structures).

**Comment by maclariz** [comment]
That should be fairly uncontroversial as it is on an alloy which is not too commercially sensitive and getting approval for publication should not be hard.  Some of the other work will need more discussion as the alloy and heat treatment conditions are commercial and we would need permission to publish from the commercial partner.

**Comment by hakonanes** [comment]
All understandable. Thank you for considering it.

We could at least make an example showing how to plot separate IPF maps of the full map shape to ease comparison, which was your issue. Any two-phase map will do in this case.

**Comment by maclariz** [comment]
I agree.  Would be nice to have the example on something where a given phase does not extend to the map edges in at least one direction, so that without doing this, you get different shaped maps.
